lib/plug/debugger.ex

```diff
@@ -180,6 +196,23 @@ defmodule Plug.Debugger do
 
     if accepts_html?(get_req_header(conn, "accept")) do
       conn = put_resp_content_type(conn, "text/html")
+      impl = Plug.Exception.impl_for(reason)
+
+      actions =
+        if Code.ensure_loaded?(impl) && function_exported?(impl, :actions, 1) do
+          reason
+          |> Plug.Exception.actions()
+          |> Enum.map(fn {action_key, metadata} ->
+            encoded_action =
+              metadata[:action]
+              |> :erlang.term_to_binary()
+              |> Base.encode64()
```

Comment:
It is time to talk about security! Instead of calling Base.encode64, call [`Plug.Crypto.MessageVerifier.sign/2`](https://github.com/elixir-plug/plug/blob/v1.0.0/lib/plug/crypto/message_verifier.ex#L14) with the `term_to_binary` result and a secret.

The secret should be generated from the `conn.secret_key_base`, something like this:

    Plug.Crypto.KeyGenerator(config.secret_key_base, "plug-debugger", cache: Plug.Keys)